libnavigation-base/src/main/java/com/mapbox/navigation/base/options/OnboardRouterOptions.kt

```diff
@@ -53,32 +51,19 @@ data class OnboardRouterOptions(
 
         /**
          * Creates a custom file path to store the road network tiles.
+         * If [filePath] is null, a default file will be created.
          */
-        fun filePath(filePath: String) =
+        fun filePath(filePath: String?) =
```

Comment:
I believe this should be non-null because if you don't specify a default file will be created but the builder "asks" you to provide one if you're interested in doing so, don't you think?

Reply:
the thought was being able to go back to the default

onboardRouterOptions.toBuilder().filePath(null).build()

but if we want to support it, best to do it explicitly with a 'useDefaultFilePath' or something

Reply:
You can always do `onboardRouterOptions.toBuilder().build()` right?